README.rst

```diff
@@ -8,7 +8,7 @@ reaching v1.0.0 in 2019. Development on Synapse and the Matrix protocol itself c
 in earnest today.
 
 Briefly, Matrix is an open standard for communications on the internet, supporting
-federation, encryption and VoIP. Matrix.org has more to say about the `goals of the
+federation, encryption, and VoIP. Matrix.org has more to say about the `goals of the
```

Comment:
Nit: the [Oxford comma](https://en.wikipedia.org/wiki/Serial_comma) is a point of style, not grammar. This is true of a few other changes too.

Reply:
so should I remove the comma or let it be there

Reply:
I don't mind either way, I'm just pointing out that this doesn't match the PR title.